fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/OpenElection.java

```diff
@@ -0,0 +1,78 @@
+package com.github.dedis.popstellar.model.network.method.message.data.election;
+
+import static com.github.dedis.popstellar.model.objects.event.EventState.CREATED;
+
+import androidx.annotation.NonNull;
+
+import com.github.dedis.popstellar.R;
+import com.github.dedis.popstellar.model.network.method.message.data.Action;
+import com.github.dedis.popstellar.model.network.method.message.data.Data;
+import com.github.dedis.popstellar.model.network.method.message.data.Objects;
+import com.google.gson.annotations.SerializedName;
+
+public final class OpenElection extends Data {
+
+  @SerializedName("lao")
+  private final String laoId;
+
+  @SerializedName("election")
+  private final String electionId;
```

Comment:
I see that you're using `@NonNull`, which is great !

But you could also use them here.
```suggestion
  @NonNull
  @SerializedName("lao")
  private final String laoId;

  @NonNull
  @SerializedName("election")
  private final String electionId;
```